python/llm/example/transformers/transformers_int4/dolly/v1/README.md

```diff
@@ -0,0 +1,78 @@
+# Dolly v1
+
+Dolly v1 models are part of the GPT-J model family, and designed for text generation tasks.
```

Comment:
This is not accurate; just remove this line.

Reply:
Have removed :)